### PR TITLE
[ADD][10.0] medical: Tour

### PR DESCRIPTION
**Pull request by lasley**
Not much here, but it sets up our touring system for medical.

* Force XML IDs for main menus
* Add Medical tour

Depends:
- [x] #34

### CODE REVIEW

**Comment by lasley** [comment]
Rebased; ready for review

File: medical/data/medical_tour.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- Make the xmlid of menus required by the Medical tour available in webclient -->
+    <record id="medical.medical_root" model="ir.ui.menu">
```
Comment (tedsalmon) [inline medical/data/medical_tour.xml]: Any idea why lint is complaining about the xml_id here being redundant? Seems like something we can ignore, just seemed odd.

**Review by tedsalmon — APPROVED** [review]
Apart from the lint complaint, LGTM!

File: medical/data/medical_tour.xml (hunk 1 above)
Comment (lasley) [reply medical/data/medical_tour.xml]: It technically is redundant with https://github.com/laslabs/vertical-medical/pull/37/files#diff-51a3344074644d0761ab5ff7e70d0b85R7

The issue is that `load_xmlid` is not a valid attribute when using the menu tag, so it has to be appended after the actual menu creation. This is the same pattern Odoo SA uses for their tours, even when the menu is located in the same module, so it seems there's not a way around this.

Comment (lasley) [reply medical/data/medical_tour.xml]: Oh wait actually I think I found it in use. Interesting, the changed that in v10

**Pull request merged**